pyproject.toml

```diff
@@ -66,6 +66,7 @@ dev = [
   "numpydoc >=1.8.0",
   "pip >=25.0",
   "pooch >=1.8.0",
+  "POT >=0.9.4",
```

Comment:
is it case-sensitive? POT here, and pot elsewhere

Reply:
I try to stick with the authority spelling, but it's a bit odd; PyPI is case-insensitive and normalizes `-` into `_`; for conda, it's case-insensitive, but dashes aren't normalized.